src/sentry/tagstore/models.py

```diff
@@ -17,6 +17,13 @@
     from sentry.tagstore.legacy.models import *  # NOQA
 elif settings.SENTRY_TAGSTORE.startswith('sentry.tagstore.v2'):
     from sentry.tagstore.v2.models import *  # NOQA
+
+    # Prevent schema migration from adding DROP TABLE for Legacy
```

Comment:
Would just doing:

```python
from sentry.tagstore import legacy
```

be enough? Or even

```python
from sentry.tagstore.legacy import models
```

Just to prevent needing to explicitly import every model.

Reply:
Oh yeah, it should, I think I did it this way for V2 because I use it in some tests.

Reply:
Or not, fixing both.